packages/perseus/src/widgets/__tests__/numeric-input.test.ts

```diff
@@ -550,24 +547,23 @@ describe("unionAnswerForms utility function", () => {
         const forms = [
             [
                 {
-                    simplify: "required",
+                    simplify: "required" as const,
                     name: "integer",
-                },
+                } as const,
             ],
             [
                 {
-                    simplify: "required",
+                    simplify: "required" as const,
                     name: "integer",
-                },
+                } as const,
             ],
         ];
 
         // act
-        // @ts-expect-error - TS2345 - Argument of type '{ simplify: string; name: string; }[][]' is not assignable to parameter of type 'readonly (readonly PerseusNumericInputAnswerForm[])[]'.
         const result = unionAnswerForms(forms);
 
         // assert
-        expect(result).toBeArrayOfSize(1);
```

Comment:
This comes from `jest-extended` but our usage of that library is so low I'm planning to remove it entirely.

https://jest-extended.jestcommunity.dev/docs/matchers/array/#tobearrayofsize

Reply:
https://khanacademy.atlassian.net/browse/LEMS-2176